suite-native/transactions/src/components/TransactionsList/TransactionIconSpinner.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import {
+    Canvas,
+    Circle,
+    vec,
+    SweepGradient,
+    useTiming,
+    useComputedValue,
+} from '@shopify/react-native-skia';
+
+import { Box } from '@suite-native/atoms';
+import { useNativeStyles, prepareNativeStyle } from '@trezor/styles';
+import { Color } from '@trezor/theme';
+
+type TransactionIconSpinnerProps = {
+    radius: number;
+    color: Color;
+};
+
+const STROKE_WIDTH = 8;
+
+const ContainerStyle = prepareNativeStyle(_ => ({
+    position: 'absolute',
+}));
+
+export const TransactionIconSpinner = ({ radius, color }: TransactionIconSpinnerProps) => {
+    const { applyStyle, utils } = useNativeStyles();
+
+    const rotation = useTiming({ from: 0, to: 100, loop: true }, { duration: 2500 });
```

Comment:
Please use Reanimated instead https://shopify.github.io/react-native-skia/docs/animations/reanimated/

We will have to migrate it soon anyway because Skia will support reanimated directly.

Reply:
fixed in [40bdfc5](https://github.com/trezor/trezor-suite/pull/7685/commits/40bdfc547bec9d4972f293bc2fa699868a835f00)